calyx-opt/src/analysis/static_par_timing.rs

```diff
@@ -14,7 +14,7 @@ type ThreadTimingMap = HashMap<u64, CellTimingMap>;
 #[derive(Default)]
 pub struct StaticParTiming {
     /// Map from par block ids to cell_timing_maps
-    cell_map: HashMap<u64, ThreadTimingMap>,
+    pub cell_map: HashMap<u64, ThreadTimingMap>,
```

Comment:
Why is this public? Should this be accessible to consumers of this pass? If so, they should not be able to mutate the field in which case we should provide an accessor method.